source/includes/_node_rpc_tx.md

```diff
@@ -0,0 +1,678 @@
+```
+
+```shell--curl
+curl $url/ \
+  -H 'Content-Type: application/json' \
+  -X POST \
+  --data '{
+    "method": "sendrawtransaction",
+    "params": [ "'$rawtx'" ]
+  }'
+```
+
+```shell--cli
+bcoin cli rpc sendrawtransaction $rawtx
+```
+
+```javascript
+const rpc = new bcoin.http.RPCClient({
+  network: 'testnet'
+});
+
+(async () => {
+  const res = await rpc.execute('sendrawtransaction', [ rawtx ]);
+
+  console.log(res);
+})().catch((err) => {
+  console.error(err.stack);
+});
+```
+
+> 
+
+```json
+"28d65fdaf5334ffd29066d7076f056bb112baa4bb0842f6eaa06171c277b4e8c"
+```
+
+Sends raw transaction without verification
+
+### Params
+N. | Name | Default |  Description
+--------- | --------- | --------- | -----------
+1 | rawtx | Required | Raw transaction hex
+
+
+
+## createrawtransaction
+
+```javascript
+let txhash, txindex, amount, address, data;
+```
+
+```shell--vars
+txhash='d1e1b6a8ff8c4d2ade2113a5dd250637e5f99667d36dcae9b70139516cb7052f';
+txindex=1;
+amount=1;
+
+address='RStiqGLWA3aSMrWDyJvur4287GQ81AtLh1';
+data='';
+```
+
+```shell--curl
+curl $url/ \
+  -H 'Content-Type: application/json' \
+  -X POST \
+  --data '{
+    "method": "createrawtransaction",
+    "params": [
+      [{ "txid": "'$txhash'", "vout": "'$txindex'" }],
+      { "'$address'": "'$amount'", "data": "'$data'" }
+    ]
+  }'
+```
+
+```shell--cli
+bcoin cli rpc createrawtransaction \
+  '[{ "txid": "'$txhash'", "vout": "'$txindex'" }]' \
+  '{ "'$address'": "'$amount'", "data": "'$data'" }'
+
+```
+
+```javascript
+const rpc = new bcoin.http.RPCClient({
+  network: 'regtest'
+});
+
+(async () => {
+  const sendTo = {
+    data: data
+  };
+
+  sendTo[address] = amount;
+
+  const res = await rpc.execute('createrawtransaction', [ [{ txid: txhash, vout: txindex }], sendTo]);
+
+  console.log(res);
+})().catch((err) => {
+  console.error(err.stack);
+});
+```
+
+> The above command returns JSON "result" like this:
+
+```json
+"01000000012f05b76c513901b7e9ca6dd36796f9e5370625dda51321de2a4d8cffa8b6e1d10100000000ffffffff0200e1f505000000001976a914c1325e8fb60bd71d23532c39b4c9e743a2cc764988ac0000000000000000026a0000000000"
+```
+
+<aside class="info">
+Note: Transaction in example doesn't spacify change output,
+you can do it by specifying another <code>address: amount</code> pair.
+</aside>
+
+Creates raw transaction without any verification and without sign.
```

Comment:
Reword for clarity: "Creates raw, unsigned transaction without any formal verification"